commit 58a7aa900bc2d9822b0d0cb596ba95a21ff0fd2d
Author: Mark Wielaard <mark@klomp.org>
Date:   Wed Nov 2 17:54:11 2022 +0100

    debuginfod: Mark extract_section function static
    
    The extract_section function in debuginfod-client.c is an internal
    function and should not be exported. Mark it as static.
    
    Signed-off-by: Mark Wielaard <mark@klomp.org>

diff --git a/debuginfod/debuginfod-client.c b/debuginfod/debuginfod-client.c
index 0c4a00cf..f48e32cc 100644
--- a/debuginfod/debuginfod-client.c
+++ b/debuginfod/debuginfod-client.c
@@ -621,7 +621,7 @@ path_escape (const char *src, char *dest)
    section name was not found.  -EEXIST indicates that the section was
    found but had type SHT_NOBITS.  */
 
-int
+static int
 extract_section (int fd, const char *section, char *fd_path, char **usr_path)
 {
   elf_version (EV_CURRENT);
